src/components/ErrorBars.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useMemo } from 'react';
+
+import { usePlotContext } from '../hooks';
+import { SeriesPointType, ErrorBarsProps } from '../types';
+import { validateAxis, validateSeriesPointError } from '../utils';
+
+interface ErrorBarsComponentProps extends ErrorBarsProps {
+  data: SeriesPointType[];
+  xAxis?: string;
+  yAxis?: string;
+}
+
+interface PointBarsProps extends Omit<ErrorBarsProps, 'hidden'> {
+  origin: { x: number; y: number };
+  top: number | null;
+  bottom: number | null;
+  left: number | null;
+  right: number | null;
+}
+
+export default function ErrorBars(props: ErrorBarsComponentProps) {
+  const { xAxis = 'x', yAxis = 'y', data, hidden, ...otherProps } = props;
+  const { axisContext } = usePlotContext();
+  const [xScale, yScale] = validateAxis(axisContext, xAxis, yAxis);
+
+  const points = useMemo(() => {
+    if (hidden) return null;
+    if ([xScale, yScale].includes(undefined)) return null;
+
+    const pointBars = data.map((point, i) => {
+      const xError = validateSeriesPointError(point.xError);
+      const yError = validateSeriesPointError(point.yError);
+      return (
+        <PointBars // eslint-disable-next-line react/no-array-index-key
+          key={`ErrorBars-${i}`}
+          origin={{ x: xScale(point.x), y: yScale(point.y) }}
+          bottom={yError ? yScale(point.y - yError[0]) : null}
+          top={yError ? yScale(point.y + yError[1]) : null}
+          left={xError ? xScale(point.x - xError[0]) : null}
+          right={xError ? xScale(point.x + xError[1]) : null}
+          {...otherProps}
+        />
+      );
+    });
+    return pointBars;
+  }, [data, xScale, yScale, hidden, otherProps]);
+  return <g>{points}</g>;
+}
+
+function PointBars(props: PointBarsProps) {
+  const {
+    origin,
+    top,
+    bottom,
+    left,
+    right,
+    style,
+    capSize = 10,
+    capStyle,
+  } = props;
+  const defaultColor = 'black';
+
+  return (
+    <g>
+      {!(top == null) && (
```

Comment:
```suggestion
      {top && (
```

Reply:
@maasencioh  I didn't do that cuz when top==0 the line should still be drawn

Reply:
The please use the `!=` operator